piet-wgsl/shader/path_coarse.wgsl

```diff
@@ -295,19 +295,22 @@ fn main(
                         tile_seg.delta = dp;
                         var y_edge = mix(lp0.y, lp1.y, (tile_x0 - lp0.x) * recip_dx);
                         if xymin.x < tile_x0 {
-                            let p = vec2(tile_x0, y_edge);
+                            let p = vec2<f32>(tile_x0, y_edge);
                             if dp.x < 0.0 {
                                 tile_seg.delta = p - lp0;
                             } else {
                                 tile_seg.origin = p;
                                 tile_seg.delta = lp1 - p;
                             }
                             if tile_seg.delta.x == 0.0 {
-                                tile_seg.delta.x = sign(dp.x) * 1e-9;
+                                tile_seg.delta.x = sign(dp.x) * 1
```

Comment:
This doesn't seem right to me. Did you mean to write out the number in unscientific notation?

Reply:
This got auto-formatted by wgsl-analyzer. It shouldn't be here